Tailscale/shared/Tailscale.sh

```diff
@@ -12,24 +15,31 @@ case "$1" in
         echo "${QPKG_NAME} is disabled."
         exit 1
     fi
-    mkdir -p -m 0755 /tmp/tailscale
```

Comment:
As of https://github.com/tailscale/tailscale/commit/7c4d017636896f027d20cd1bb8f65b03b5d35d7e both `tailscale` and `tailscaled` set their local socket path to /tmp/tailscale/tailscale.sock.

Even after moving the PID file to /var/run, I would think this mkdir is still needed.

Reply:
I have never had issues in my environment with not creating /tmp/tailscale location. But I don't see any issues leaving it in.